backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java

```diff
@@ -66,9 +66,15 @@ public DockerCassandraRule() {
                     "device", "tmpfs",
                     "o", "size=100m"));
         deleteTmpsFsCmd = client.removeVolumeCmd(tmpFsName);
-        boolean deleteOnExit = false;
-        cassandraContainer = new GenericContainer<>(
-            new ImageFromDockerfile("cassandra_2_2_10", deleteOnExit)
+        cassandraContainer = new CassandraContainer(tmpFsName);
+    }
+
+    private static class CassandraContainer extends GenericContainer<CassandraContainer> {
+
+        private static final boolean DELETE_ON_EXIT = false;
+
+        public CassandraContainer(String tmpFsName) {
```

Comment:
It makes the reader think (s)he can instantiate it several time but that's not the case.